src/main/java/beam/analysis/plots/RideHailWaitingAnalysis.java

```diff
@@ -216,6 +217,13 @@ public void createGraph(IterationEndsEvent event) throws IOException {
         writeRideHailWaitingIndividualStatCSV(event.getIteration());
     }
 
+    @Override
+    public Map<String, Double> getSummaryStats(){
+        Map<String, Double> averageRidehailWaitTime = new HashMap();
```

Comment:
How about just one lining this:

```
return new HashMap<String, Double>() {{
    put("averageRidehailWaitTime" , waitTimeSum / rideHailCount);
}};
```

Reply:
Hi @JustinPihony, I have made this change.